code/modules/pda/PDA.dm

```diff
@@ -160,8 +160,12 @@ GLOBAL_LIST_EMPTY(PDAs)
 	. = ..()
 	if(id)
 		. += image('icons/goonstation/objects/pda_overlay.dmi', id.icon_state)
+
 	if(length(notifying_programs))
-		. += image('icons/obj/pda.dmi', "pda-r")
+		if(icon_state == "pda-library")
```

Comment:
Is it not better to use a method other than icon_state? This can become an oversight in the future.

Reply:
I just don't see a way to make it better